TodoList/Extensions/UITableViewExtensions.swift

```diff
@@ -2,21 +2,30 @@ import UIKit
 
 extension UITableView {
     
+    /// Registers a nib object containing a cell with the table view. Use this function to avoid creating a reuse ID.
+    ///
+    /// - Parameters:- cellType: Name of xib to register
     func register<T: UITableViewCell>(cellType: T.Type) {
         let reuseIdentifier = String(describing: T.self)
         let nib = UINib(nibName: reuseIdentifier, bundle: Bundle(for: T.self))
         
         register(nib, forCellReuseIdentifier: reuseIdentifier)
     }
     
+    /// Returns a reusable table-view cell object and adds it to the table. Use this function to avoid creating a reuse ID.
+    ///
+    /// - Parameter for: The index path specifying the location of the cell. The data source receives this information when
+    ///     it is asked for the cell and should just pass it along. This method uses the index path to
+    ///     perform additional configuration based on the cell’s position in the table view.
+    ///
+    /// - Returns: A UITableViewCell object with the associated reuse identifier. This method always returns a valid cell.
```

Comment:
This is very good but I would modify slightly: `A generic UITableViewCell with the associated "reuseIdentifier"`.
I would also explain what happens if it couldn't find a cell with that identifier.